02-spatial-data.Rmd

```diff
@@ -212,29 +224,33 @@ For more advanced map making we recommend using a dedicated visualization packag
 - map export 
 -->
 
-<!-- Maybe show also somewhere that `world[0]` produces only a plot of the geometry which is rather useful if you do not want to plot a specific attribute. This way, you can for example dismiss the col = "white"-argument in your Nigeria example. -->
+<!-- Maybe show also somewhere that `world[0]` produces only a plot of the geometry which is rather useful if you do not want to plot a specific attribute. This way, you can for example dismiss the col = "white"-argument in your Nigeria example.
```

Comment:
Yes - please do mention the `plot(x[0])` behaviour in the prose: interesting and potentially useful to people who like ultra-concise code.